### DISCUSSION TITLE
Could we merge with the other fork?

### DISCUSSION THREAD

**Discussion by ghost** [tenacityteam/tenacity-legacy#114] [Ideas]
Tenacity has had a rocky start and an even worse time, and I just had the thought of migration to Sneedacity, which actually appears to be going well.

**Comment by ghost** [comment]
This is genuine. I want to know what we think.

**Comment by bpepple** [comment]
> Tenacity has had a rocky start and an even worse time, and I just had the thought of migration to Sneedacity, which actually appears to be going well.

The only problem seems to be the 4chan dipshits attempting to flood the discussions with false accusations.

**Reply by ghost** [reply]
> > Tenacity has had a rocky start and an even worse time, and I just had the thought of migration to Sneedacity, which actually appears to be going well.
> 
> The only problem seems to be the 4chan dipshits attempting to flood the discussions with false accusations.

oh?

**Comment by differentprogramming** [comment]
Welcome to github, 4chan.

**Comment by drankinatty** [comment]
Beware of Greeks bearing gifts..... Stewardship of this fork is on firm footing. The actual sound FOSS philosophy is evidenced by the repugnant data collection and tracking code removed. I see no reason for involvement of any other project. The band of talented individuals that has taken the fork this far are more than capable of managing things from here. If you run low on manpower, I'm happy to help. Good job.

**Comment by caughtquick** [comment]
> Tenacity has had a rocky start and an even worse time, and I just had the thought of migration to Sneedacity, which actually appears to be going well.

No. While the repo has similar goals we will not collaborate on a project with the same people (group of people) that doxxed the original maintainer of this project. We will do just fine without them.

**Comment by ErikPrantare** [comment]
Sneedacity devs pushed breaking code by substituting audacity with sneedacity in the whole project. Utter buffoonery lmao.

**Comment by Flyingmana** [comment]
What I saw, some of the changes added to Sneedacity, could be seen as violation of the projects Code of Conduct.
![image](https://user-images.githubusercontent.com/237319/124953242-4155a900-e015-11eb-9fad-fb8e8bdeaf04.png)


Alone their changes to their "code of conduct" are a big argument against merging or even working together.

**Reply by falkTX** [reply]
they also updated the copyright headers, while they don't own the 2020 copyright for sure.

**Comment by Semisol** [comment]
We are not merging with a fork maintained by people against us. But if people like panda (you should have seen them in the IRC) which didn't troll want to come here and contribute we are fully fine. Locking.
